src/d3a/models/strategy/pv.py

```diff
@@ -93,10 +96,11 @@ def event_activate(self):
     def _incorporate_rate_restrictions(self, initial_sell_rate, current_time):
         energy_rate = max(initial_sell_rate, self.final_selling_rate)
         rounded_energy_rate = round(energy_rate, 2)
-        if rounded_energy_rate == 0.0:
-            # Initial selling offer
-            rounded_energy_rate =\
-                self.area.config.market_maker_rate[current_time]
+        # TODO: Find out the reason for this block, at the moment it does not make sense
```

Comment:
My guess is that it is one of the last traces of the Brainbot implementation. There probably was a convention where if the initial energy rate configuration of the PV was 0, the market maker rate was used instead, that's why this still exists up to this day. I would even go ahead and remove this completely, but if you want to play it safe leave it as is.

Reply:
Yes, I am always in favour of removing code. ;-) 
So will do.